index.html

```diff
@@ -123,30 +144,30 @@ <h2 class="about-title">About <br> Myself</h2>
                 <h6 class="des">Hello I’m a software developer! I can help you build a product , feature or website Look
                     through some of my work and experience! If you like what you see and have a project you need coded,
                     don’t hestiate to contact me.</h6>
+                <div class="Right contact">
+                    <h5 class="media-text bottom">LET’S CONNECT</h5>
+
+                    <ul class="right-icons bottom-list">
+                        <li><a href="https://twitter.com/Ernestbore3" target="_blank" class="media-link"
+                                rel="noopener"><i class="fab fa-twitter"></i></a></li>
+                        <li><a href="https://www.linkedin.com/in/ernest-bore-3b6042211/" target="_blank"
+                                class="media-link" rel="noopener"><i class="fab fa-linkedin"></i></a></li>
+                        <li><a href="https://medium.com/" target="_blank" class="media-link" rel="noopener"><i
+                                    class="fab fa-medium-m"></i></a></li>
+                        <li><a href="https://github.com/ernestkorir" target="_blank" class="media-link"
+                                rel="noopener"><i class="fab fa-github"></i></a></li>
+                        <li><a href="https://angel.co/ernestkorir" target="_blank" class="media-link" rel="noopener"><i
+                                    class="fab fa-angellist"></i></a></li>
+                    </ul>
+                </div>
+                <button class="resume">Get my resume</button>
```

Comment:
[OPTIONAL]
- [ ] Although overlooked by the previous reviewer, I highly recommend you consider adjusting the space between these two elements. `You can use display: flex; justify-content:space-around`